sys/shell/commands/sc_ccnl.c

```diff
@@ -170,10 +167,16 @@ int _ccnl_interest(int argc, char **argv)
         addr_len = gnrc_netif_addr_from_str(relay_addr, sizeof(relay_addr), argv[2]);
     }
 
+    memset(_int_buf, '\0', BUF_SIZE);
+    memset(_cont_buf, '\0', BUF_SIZE);
```

Comment:
For `_int_buf` it's not a problem, because it will get initialized with the same value during the loop anyway. For _cont_buf` it's also okay, before it will only be written if `ccnl_wait_for_chunk()` returns with a positive value.